website/pages/en/arbitrum/l2-transfer-tools-faq.mdx

```diff
@@ -38,55 +48,143 @@ To transfer your subgraph, you will need to complete the following steps:
 
 After the 20 minute transfer window, you will need to confirm the transfer with a transaction in the UI to finish the transfer, but the transfer tool will guide you through this. Your L1 endpoint will continue to be supported during the transfer window and a grace period after. It is encouraged that you update your endpoint when convenient for you.
 
-## Will there be a down-time to my endpoint while re-publishing?
+### Will there be a down-time to my endpoint while re-publishing?
 
 There should be no down time when using the transfer tool to move your subgraph to L2.Your L1 endpoint will continue to be supported during the transfer window and a grace period after. It is encouraged that you update your endpoint when convenient for you.
 
-## Is publishing and versioning the same on L2 as Ethereum Ethereum mainnet?
+### Is publishing and versioning the same on L2 as Ethereum Ethereum mainnet?
 
-Yes. Be sure to select Arbitrum One as your published network when publishing in Subgraph Studio. In the Studio, the latest endpoint will be available which points to the latest updated version of the subgraph.
+Yes. Select Arbitrum One as your published network when publishing in Subgraph Studio. In the Studio, the latest endpoint will be available which points to the latest updated version of the subgraph.
 
-## Will my subgraph's curation move with my subgraph?
+### Will my subgraph's curation move with my subgraph?
 
 If you've chosen auto-migrating signal, 100% of your own curation will move with your subgraph to Arbitrum One. All of the subgraph's curation signal will be converted to GRT at the time of the transfer, and the GRT corresponding to your curation signal will be used to mint signal on the L2 subgraph.
 
 Other Curators can choose whether to withdraw their fraction of GRT, or also transfer it to L2 to mint signal on the same subgraph.
 
-## Can I move my subgraph back to Ethereum mainnet after I transfer?
+### Can I move my subgraph back to Ethereum mainnet after I transfer?
 
 Once transferred, your Ethereum mainnet version of this subgraph will be deprecated. If you would like to move back to mainnet, you will need to redeploy and publish back to mainnet. However, transferring back to Ethereum mainnet is strongly discouraged as indexing rewards will eventually be distributed entirely on Arbitrum One.
 
-## Why do I need bridged ETH to complete my transfer?
+### Why do I need bridged ETH to complete my transfer?
 
 Gas fees on Arbitrum One are paid using bridged ETH (i.e. ETH that has been bridged to Arbitrum One). However, the gas fees are significantly lower when compared to Ethereum mainnet.
 
+## Delegation
+
+### How do I transfer my delegation?
+
+<VideoEmbed youtube="CAAQCKw5eV4" />
+
+To transfer your delegation, you will need to complete the following steps:
+
+1.  Initiate delegation transfer on Ethereum mainnet
+
+2.  Wait 20 minutes for confirmation
+
+3.  Confirm delegation transfer on Arbitrum
```

Comment:
Curious about the formatting of this list. While this is valid Markdown and it will render a single list, the blank lines between the items and the extra whitespace between the markers and the text are not necessary:

```suggestion
1. Initiate delegation transfer on Ethereum mainnet
2. Wait 20 minutes for confirmation
3. Confirm delegation transfer on Arbitrum
```

But if you use a Markdown editor that formats lists this way, or find that it's more readable, feel free to ignore me. Prettier doesn't seem to care either way. 😛

Reply:
Thank you! Happy to condense!